tensorflow_addons/text/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""Text-processing ops."""
+"""Additional text-processing ops."""
```

Comment:
My minor concern is that crf in `text` is more like a probabilistic model rather than a text-processing operation.

Reply:
Agree. I'm thinking CRF will should be moved to Layers. Individual CRF functions can still be called but they'll probably be marked private.